android/src/main/java/com/reactlibrary/RNDfpBannerViewManager.java

```diff
@@ -189,17 +194,17 @@ public void setAdSizes(final ReactViewGroup view, final ReadableArray adSizesPro
       AdSize[] adSizes = adSizesArrayList.toArray(new AdSize[adSizesArrayList.size()]);
 
       attachNewAdView(view);
-      PublisherAdView newAdView = (PublisherAdView) view.getChildAt(0);
-      newAdView.setAdSizes(adSizes);
-      newAdView.setAdUnitId(adUnitID);
+      adView = (PublisherAdView) view.getChildAt(0);
```

Comment:
If this is dynamic this could get unsafe. Perhaps before casting we could assign `getChildAt(0)` to a `View` then use `instanceof PublisherAdView` to safely check. However in that case we'd likely just return out of the method.